examples/ecommerce/src/App.vue

```diff
@@ -482,8 +482,10 @@ export default {
     },
     closeFilters() {
       document.body.classList.remove('filtering');
-      const resultsContainer = document.querySelector('.container-results');
-      resultsContainer.scrollIntoView();
+      if (!this.resultsContainer) {
```

Comment:
I meant storing it in the `script` tag after the imports.

Reply:
It won't successfully query the element until mounted. We can move it to `mounted` lifecycle method. What do you think?

Reply:
Ah right – I'm not proficient with Vue so do as seems good to you!

Reply:
a4e3e68